c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -6,6 +6,12 @@ Version 1.6
 
 1. Get the new and heavily modified files:
 
+    first follow the instruction as explained in the following documentation
+    chapter:
+    http://docs.camptocamp.net/c2cgeoportal/1.6/integrator/upgrade_application.html#easy-upgrading-an-application-from-1-5-to-1-6
```

Comment:
I didnt want to renumber all the points, but if you think it's better, we
can do it

On Wed, Jul 1, 2015 at 9:47 AM, Stéphane Brunner <notifications@github.com>
wrote:

> In c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl
> <https://github.com/camptocamp/c2cgeoportal/pull/1428#discussion_r33655407>
> :
>
> > @@ -6,6 +6,12 @@ Version 1.6
> >
> >  1. Get the new and heavily modified files:
> >
> > +    first follow the instruction as explained in the following documentation
> > +    chapter:
> > +    http://docs.camptocamp.net/c2cgeoportal/1.6/integrator/upgrade_application.html#easy-upgrading-an-application-from-1-5-to-1-6
>
> Shouldn't be put outside the step 1 ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/camptocamp/c2cgeoportal/pull/1428/files#r33655407>.
>